src/NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp

```diff
@@ -288,4 +288,44 @@ struct DerivCompute
       tmpl::list<VariablesTag, domain::Tags::Mesh<Dim>, InverseJacobianTag>;
 };
 
+/*!
+ * \ingroup DataBoxTagsGroup
+ * \brief Computes the spatial derivative of a single tensor tag not in a
+ * Variables.
+ *
+ * Computes the spatial derivative for a single tensor represented by
+ * 'TensorTag' in the frame mapped to by 'InverseJacobianTag'. It takes a
+ * single Tensor designated by 'TensorTag', the inverse Jacobian, and a mesh
+ * can be passed in but is not required.
+ *
+ * This tag may be retrieved via '::Tags::DerivTensorCompute<TensorTag,
```

Comment:
This last clarification isn't necessary here. All our normal compute tags behave this way. It's just the `Variables` ones that are a bit weird